summit/app/index.html

```diff
@@ -67,6 +68,7 @@
               <iron-selector class="wide-only" selected="[[page]]" activate-event="click" attr-for-selected="name">
                 <a name="schedule" class="tab" href="/summit/schedule">SCHEDULE</a>
                 <a name="speakers" class="tab" href="/summit/speakers">SPEAKERS</a>
+                <a name="codelabs" class="tab" href="/summit/codelabs" target="_blank">CODELABS</a>
```

Comment:
Is this going to pop open a new tab but also cause the router to change routes in the previous tab?